documents/store-operations/bopis/order-details-page/README.md

```diff
@@ -11,26 +11,47 @@ Users can access the Order Details page by clicking on any order card from any o
 
 On clicking on any order card in the `Open`, `Packed` or `Completed` tab, users will be directed to the order details page. The order details page displays key details of an order and also enables store associates to perform the certain functions.
 
-### Reject Orders
+### Item Details
+
+At the top of the Order Details page, this section displays the order name and order ID. It lists all items in order, along with their product image, primary, and secondary identifiers. If an item is part of a kit, a "KIT" tag is shown next to the item name. Each item has two icons: a box icon that shows the quantity on hand and a bin icon to reject the item. 
 
-During peak hours or promotional events, brick-and-mortar retail locations often face increased in-store traffic, heightening the risk that the requested item for pick-up may become unavailable due to in-store purchases. In such instances, store associates may need to reject an order or reject an order partially in case the order item inventory is insufficient at the store. Partial order rejection feature can be enabled using the toggle on the settings page.
+The action buttons shown depend on the order status. When the order is in Open status, buttons like Ready for Pickup and Reject Items are available. After the order is marked Ready for Pickup, these buttons change to Cancel Item and Handover. If the order is marked as Ready for Pickup, the only action available is to cancel it. If an item is damaged or out of stock, it should be rejected before packing. After packing, the order is marked Ready for Pickup, and an email is sent to the customer. So, unpacking or rejecting is not allowed at this stage.
 
-To reject an entire order, click on the order card in the `Open` tab to open the order details page. Click on `Reject Order` to reject the order, select an appropriate reason for rejecting the order in the pop-up window that appears and click on the `save` icon. Another pop-up will appear to confirm order rejection, select `Reject` to reject the order.
+Selecting Cancel Item opens a pop-up displaying a list of cancellation reasons. When an item is cancelled, an email is sent to the customer. If items are rejected, a notification email is also sent, and the order may be re-routed. For more information, refer to the Order Re-Routing App User Manual. 
```

Comment:
I don't think any button is labeled "Cancel Item"

Reply:
You have mixed cancel and reject into the same sentence. Ideas need to be logically grouped for the reader to consume them